package.json

```diff
@@ -17,6 +17,14 @@
   },
   "homepage": "https://github.com/oppia/oppia#readme",
   "devDependencies": {
+    "@angular/common": "^8.0.0",
```

Comment:
To confirm: you ran npm audit?

Reply:
Yes I ran it (and also Kevin confirmed it)